samples/tfm_integration/psa_level_1/prj.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_LOG_DEFAULT_LEVEL=4
 #CONFIG_PSA_SHELL=y
 
 CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_IPC=y
```

Comment:
I am fine with this, but i think we really need to change the sample name, since level_1 refers, i guess to the isolation level? and with IPC we can have any level, basically...

Reply:
I agree ... the name has bothered me for a while. I'll try to come up with something more appropriate with PSA since it's basically showing how to use the various PSA APIs.

Reply:
@ioannisg @oyvindronningstad @urutva What do you think of renaming this `psa_crypto` since the main goal of the demo is to show off how to use the PSA Crypto API? The level 1 label really no longer has any meaning to this sample as is.

The documentation would also need to be updated anywhere `psa_lvl_1` is present, of course, but I think that still fits in the intention of this PR to improve this sample.